langtest/transform/robustness.py

```diff
@@ -491,6 +491,8 @@ def transform(
 
                 ent_start_pos = [1 if label[0] == "B" else 0 for label in sample_labels]
                 ent_idx = [i for i, value in enumerate(ent_start_pos) if value == 1]
+                if not ent_idx:
+                    continue
 
```

Comment:
I'm not sure how is this possible?
We do a check a few lines above that not all tokens are "O" -> so there should be at least one "B" right?

Reply:
Yes, you are right. We can remove it